src/main/java/org/broadinstitute/consent/http/resources/DataSetResource.java

```diff
@@ -226,11 +226,11 @@ public Response createDataSet(
     @GET
     @Produces("application/json")
     @PermitAll
-    public Response describeDataSets(@Context HttpServletRequest request , @QueryParam("dacUserId") Integer dacUserId){
+    public Response describeDataSets(@Context HttpServletRequest request, @QueryParam("dacUserId") Integer dacUserId){
         if (StringUtils.isEmpty(request.getParameter("dacUserId"))) {
             return Response.status(Response.Status.NOT_FOUND).build();
         } else {
-            Collection<DataSetDTO> dataSetList = api.describeDataSets(dacUserId);
+            Collection<DataSetDTO> dataSetList = datasetService.describeDatasets(dacUserId);
```

Comment:
Went ahead and replaced the old describeDatasets as well, since it should have the same filtering logic (without a partial string to filter by)

Reply:
The dataset catalog calls this endpoint. According to the underlying logic in the service class:
```
Admins => see all datasets, active and inactive
Chairs => see all active datasets and inactive ones that are part of the user's DAC
Others => see all active datasets
```
This is correct, as far as it stands. The issue I see now is in the UI. It will need to be able to display the dataset correctly. For example, I, as both a researcher and an admin on dev, should be able to see disabled datasets, but should not be able to select them for creating a DAR. Let's follow up to ensure we have a story that covers this.

Reply:
Follow up. The UI code is definitely impacted by this. [Here is the current code](https://github.com/DataBiosphere/duos-ui/blob/develop/src/pages/DatasetCatalog.js#L426) that doesn't have any logic around the validity of the dataset selection.

Reply:
Ok! Just to clarify, we want to disable the selection checkbox if a dataset is inactive/disabled?

Reply:
<img width="704" alt="Screen Shot 2021-01-05 at 2 36 35 PM" src="https://user-images.githubusercontent.com/43456581/103690954-71879680-4f63-11eb-91ea-bef9adca35a3.png">
Also, we might want to flip the enable/disable icons... kind of confusing to click cancel to enable, now that I think about it.

Reply:
Ticket created here: https://broadworkbench.atlassian.net/browse/DUOS-901